tests/tasks/lib/test_mnli.py

```diff
@@ -1,34 +1,321 @@
+            "guy",
+            "from",
+            "triple",
+            "A",
+            "then",
+            "a",
+            "double",
+            "A",
+            "guy",
+            "goes",
+            "up",
+            "to",
+            "replace",
+            "him",
+            "and",
+            "a",
+            "single",
+            "A",
+            "guy",
+            "goes",
+            "up",
+            "to",
+            "replace",
+            "him",
+        ],
+        "hypothesis": [
+            "You",
+            "lose",
+            "the",
+            "things",
+            "to",
+            "the",
+            "following",
+            "level",
+            "if",
+            "the",
+            "people",
+            "recall.",
+        ],
+        "label_id": 1,
+    },
+    {
+        "guid": "train-2",
+        "premise": [
+            "One",
+            "of",
+            "our",
+            "number",
+            "will",
+            "carry",
+            "out",
+            "your",
+            "instructions",
+            "minutely.",
+        ],
+        "hypothesis": [
+            "A",
+            "member",
+            "of",
+            "my",
+            "team",
+            "will",
+            "execute",
+            "your",
+            "orders",
+            "with",
+            "immense",
+            "precision.",
+        ],
+        "label_id": 1,
+    },
+    {
+        "guid": "train-3",
+        "premise": [
+            "How",
+            "do",
+            "you",
+            "know?",
+            "All",
+            "this",
+            "is",
+            "their",
+            "information",
+            "again.",
+        ],
+        "hypothesis": ["This", "information", "belongs", "to", "them."],
+        "label_id": 1,
+    },
+    {
+        "guid": "train-4",
+        "premise": [
+            "yeah",
+            "i",
+            "tell",
+            "you",
+            "what",
+            "though",
+            "if",
+            "you",
+            "go",
+            "price",
+            "some",
+            "of",
+            "those",
+            "tennis",
+            "shoes",
+            "i",
+            "can",
+            "see",
+            "why",
+            "now",
+            "you",
+            "know",
+            "they're",
+            "getting",
+            "up",
+            "in",
+            "the",
+            "hundred",
+            "dollar",
+            "range",
+        ],
+        "hypothesis": ["The", "tennis", "shoes", "have", "a", "range", "of", "prices."],
+        "label_id": 2,
+    },
+]
+
+FEATURIZED_TRAIN_EXAMPLE_0 = {
+    "guid": "train-0",
+    "input_ids": np.array([1, 4, 5, 6, 7, 8, 9, 10, 11, 2, 15, 13, 16, 17, 18, 19, 5, 6, 20, 2]),
+    "input_mask": np.array([1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]),
+    "segment_ids": np.array([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]),
```

Comment:
@zphang already merged but do `input_ids, input_mask, segment_ids` look correct for this example?